src/modules/events.js

```diff
@@ -0,0 +1,202 @@
+    data.forEach((element) => {
+      if (element.show.image.original !== null) {
+        const { id, name, image } = element.show;
+        let count = 0;
+        if (loveData.length > 0) {
+          loveData.forEach((element) => {
+            if (element.item_id === id) {
+              count = element.likes;
+            }
+          });
+        }
+        const container = `
+                    <div class="items">
+                      <img class="poster" src="${image.original}" alt="#" />
+                      <h2 class="head-title">${name}</h2>
+                      <div class="likes_love">
+                        <span id=${id}>Likes: <span class="innerSpan" >${count}</span> </span>
+                        <button class="love"><img id=${id} src="${love}"></button>
+                      </div>
+                      <div class="comment_reserve">
+                        <button type="button" data-id="${id}" showType=${element.show.type} lang = ${element.show.language}  runtime = ${element.show.runtime} premiered = ${element.show.premiered} class="cmnt">Comments</button>
+                        <button type="button" class="rsrv">Reservation</button>
+                      </div>
+                    </div>
+                  `;
+        mainElement.insertAdjacentHTML('beforeend', container);
+        const total = loveCounter.countItems();
+        totals.textContent = `${total}`;
+      }
+    });
+  };
+
+  handleLikes = () => {
+    const love = document.querySelectorAll('.love');
+    love.forEach((element) => {
+      element.addEventListener('click', (event) => {
+        const id = event.target.getAttribute('id');
+        loveCounter.countLove(event, id);
+        tvShowApi.sendLoveData(id);
+      });
+    });
+  }
+
+  // This function will handle the click event of comment button
+  handleCommentBtnClick = () => {
+    const commentBtn = document.querySelectorAll('.cmnt');
+    commentBtn.forEach((btn) => {
+      btn.addEventListener('click', (event) => {
+        this.showPopup(event);
+      });
+    });
+  };
+
+  // This function will create a new container for comment popUp and asign it into the mainElement
+  showPopup = (event) => {
+    const item = event.target.closest('.items');
+    const id = event.target.getAttribute('data-id');
+    const showImageSrc = item.querySelector('img').getAttribute('src');
+    const name = item.querySelector('h2').textContent;
+    const type = event.target.getAttribute('showType');
+    const language = event.target.getAttribute('lang');
+    const premiered = event.target.getAttribute('premiered');
+    const duration = event.target.getAttribute('runtime');
+    const commentPopup = `
+        <section class="popupContainer">
+            <div class="popupSection">
+                <div class="imgContainer">
+                    <div class="imgComment">
+                    <img id="showImg" src="${showImageSrc}" alt="">
```

Comment:
- [ Optional ] I think it would be better if the poster image is not so big that it prevents the user from seeing the _Add Comments form_. I think if the image is as big as it is now a user might not know there is a form below the image. I had to scroll two screens to get to the comments form, please see the screenshot below for more details on this.

## Screenshot
![Screenshot from 2023-06-03 05-11-11](https://github.com/ibugithub/JavaScript_Group_Capstone/assets/16392046/f2e0c60f-f308-4696-b618-c597115f1246)

![Screenshot from 2023-06-03 05-13-09](https://github.com/ibugithub/JavaScript_Group_Capstone/assets/16392046/c02a7f4a-5939-469d-b432-aba514395960)

![Screenshot from 2023-06-03 05-14-39](https://github.com/ibugithub/JavaScript_Group_Capstone/assets/16392046/e576469a-14ea-403c-96a7-b4a4b8196a6b)

Reply:
Fix the image width.